purchases_ui_flutter/lib/views/paywall_footer_view.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:purchases_flutter/models/offering_wrapper.dart';
+
+import 'internal_paywall_footer_view.dart';
+
+/// View that displays the paywall in footer mode.
+/// Not supported in macOS currently.
+///
+/// [offering] (Optional) The offering object to be displayed in the paywall.
+/// Obtained from [Purchases.getOfferings].
+///
+/// [content] The content to be displayed above the paywall. Make sure you
+/// apply a padding to the bottom of your content to avoid overlap. This padding
+/// can be obtained from [PaywallFooterView.roundedCornerRadius].
```

Comment:
Is there a way we can add the padding ourselves? In React Native we do add it

Reply:
We could add it, but then it would be much more difficult to extend the top view below the borders... Mentioned the alternative in this comment: https://github.com/RevenueCat/purchases-flutter/pull/996#discussion_r1512663338